src/_locales/en_US/messages.json

```diff
@@ -355,6 +355,10 @@
         "message": "Donate to EFF",
         "description": "A button on the intro page, and a link in the popup footer."
     },
+    "review_pb": {
```

Comment:
Should move this to the popup section to where it sort of logically appears in the popup, and prefix the ID with "popup_".

Reply:
(The donate text is reused on the welcome page and the welcome page comes first in the translation doc so that's why the donate text is where it is.)